### DISCUSSION TITLE
Why support does not reply ?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#140849] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

It has been 5 days and support does not reply. How much time will it take ?

**Comment by davevad93** [comment]
Hi @FeridQ ,

Support can take a long time to reply because they have to deal with a lot of daily requests, so I suggest you to be patient because it can take a while, even weeks.

**Comment by ebndev** [comment]
Hi @FeridQ,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
